style.css

```diff
@@ -304,4 +309,382 @@ a {
     justify-content: center;
     margin-left: -42px;
   }
+
+  .chat {
+    display: none;
+  }
+
+  .form-2 {
+    display: none;
+  }
+
+  .log-2 {
+    display: none;
+  }
+
+  .footer-hr-2 {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .mobile {
+    background-image: url(/images/twocirclesheader.svg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: 0 150px;
+  }
+
+  .toolbar {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+  }
+
+  .john {
+    font-size: 18px;
+    color: #fff;
+  }
+
+  .menu-list {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    column-gap: 50px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .about h1 {
+    font-size: 90px;
+    line-height: 105px;
+    color: #9390ff;
+    margin-bottom: 0%;
+  }
+
+  .about h2 {
+    font-size: 40px;
+    line-height: 47px;
+    color: #babbcd;
+  }
+
+  .about .first-para {
+    font-size: 20px;
+    line-height: 32px;
+    color: #9196b0;
+    width: 447px;
+    margin: 20px 400px;
+    text-align: center;
+  }
+
+  .divider {
+    width: 35%;
+    border: 1px solid #646369;
+  }
+
+  .known {
+    display: flex;
+    margin: 0 450px;
+    justify-content: center;
+    margin-left: 408px;
+  }
+
+  .certifications {
+    display: flex;
+    justify-content: center;
+  }
+
+  h5 {
+    color: #7571d8;
+    font-size: 18px;
+  }
+
+  .base li {
+    border-right: 2px solid #555;
+    padding-right: 10px;
+    height: 30px;
+    padding-top: 10px;
+    color: #ffff;
+  }
+
+  .frameworks h5 {
+    margin-left: 40px;
+  }
+
+  .language {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .language ul {
+    margin-top: 0;
+  }
+
+  .language ul li {
+    color: #f5f5ff;
+    font-size: 15px;
+    line-height: 24px;
+    margin-top: 8px;
+    margin-left: -40px;
+  }
+
+  .frameworks {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-right: 40px;
+  }
+
+  .frameworks ul {
+    margin-top: 0;
+  }
+
+  .frameworks ul li {
+    color: #f5f5ff;
+    font-size: 15px;
+    line-height: 24px;
+    margin-top: 8px;
+    margin-left: 0;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 80%;
+    margin: 0 auto;
+    margin-bottom: 20px;
+  }
+
+  .skills ul {
+    margin-top: 0;
+  }
+
+  .skills ul li {
+    color: #f5f5ff;
+    font-size: 15px;
+    line-height: 24px;
+    margin-top: 8px;
+    margin-left: -40px;
+  }
+
+  .logos {
+    display: flex;
+    justify-content: space-around;
+    margin-top: 50px;
+    margin-bottom: 150px;
+  }
+
+  .logos ul {
+    display: flex;
+    margin-left: -41px;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    background-image: url(/images/Group65.svg);
```

Comment:
- [ ] Same here please add `./` before URL path because your background image not working online maybe it's working locally but online it's not.
You can do it like that: `background-image: url(./images/Group65.svg);`